library/src/main/java/ca/rmen/sunrisesunset/SunriseSunset.java

```diff
@@ -659,7 +659,7 @@ public static long getDayLength(Calendar calendar, double latitude, double longi
 				}
 			}
 		}
-		return (sunriseSunset[1].getTimeInMillis() - sunriseSunset[0].getTimeInMillis()) / 1000;
+		return sunriseSunset[1].getTimeInMillis() - sunriseSunset[0].getTimeInMillis();

```

Comment:
Thanks for catching this!

The two `return SECONDS_IN_DAY` cases above should be adapted to return a value in milliseconds too. Since that constant is only used here, it can be changed to `MILLISECONDS_IN_DAY`. This change will break one of the unit tests which will need to be adapted to check for a day length in ms. (`testAntarctica(ca.rmen.sunrisesunset.test.SunriseSunsetSpecificLocationsTest)`).

Let me know if you'd like to make these changes, otherwise I can go ahead and do it :)

Thanks!

Reply:
I’m not familiar with your library, so it’s probably better when you do the other changes.

Reply:
Ok :)